services/galley/src/Galley/API/Internal.hs

```diff
@@ -72,7 +72,7 @@ import Polysemy
 import Polysemy.Error
 import Polysemy.Input
 import Polysemy.TinyLog qualified as P
-import Servant hiding (JSON, WithStatus)
+import Servant hiding (JSON)
```

Comment:
```suggestion
import Servant
```

Reply:
(I expanded all of the things and it doesn't appear as though there's any other use of `JSON`)

Reply:
Fixed